cpp/tests/io/nested_json_test.cpp

```diff
@@ -0,0 +1,508 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cudf_test/column_wrapper.hpp"
+
+#include <io/json/nested_json.hpp>
+#include <io/utilities/hostdevice_vector.hpp>
+
+#include <cudf/io/datasource.hpp>
+#include <cudf/io/parquet.hpp>
+
+#include <cudf_test/base_fixture.hpp>
+#include <cudf_test/column_utilities.hpp>
+#include <cudf_test/cudf_gtest.hpp>
+#include <cudf_test/io_metadata_utilities.hpp>
+#include <cudf_test/table_utilities.hpp>
+
+#include <cudf/lists/lists_column_view.hpp>
```

Comment:
Sort with other `cudf` includes.

Reply:
Whoa, I thought the style checks caught this these days?

Reply:
No. clang-format will alphabetize the includes within a section, but we do not have any tooling that enforces that includes are separated by package. That's just described in the dev guide.

https://github.com/rapidsai/cudf/blob/branch-22.10/cpp/docs/DEVELOPER_GUIDE.md#includes